CMake/DCMTKConfig.cmake.in

```diff
@@ -110,6 +110,19 @@ SET_AND_CHECK(DCMTK_TARGETS "@PACKAGE_DCMTK_CMKDIR_CONFIG@/DCMTKTargets.cmake")
 # Compatibility: This variable is deprecated
 set(DCMTK_INCLUDE_DIR ${DCMTK_INCLUDE_DIRS})
 
+# Depend packages
+if(DCMTK_WITH_ZLIB)
+  if(NOT ZLIB_FOUND)
+    find_package(ZLIB)
```

Comment:
You should use find_dependency here for better error handling.
https://stackoverflow.com/questions/50263280/difference-between-find-dependency-and-find-package-in-cmake

Reply:
thanks, did not know about this one yet